app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+helper_method :current_user
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id].present?
```

Comment:
`find_by`

Reply:
a dlaczego właściwie find_by? Wyczytałem, że find zwykle używa się do zwracania pojedynczego rzędu zwykle po id tak jak w moim przypadku, a żeby odsłużyć find_by potrzebuje pary key/value więc nawet nie wiem co mogłoby to być w tym przypadku?